yaml_cpp_vendor-extras.cmake.in

```diff
@@ -13,6 +13,11 @@
 # endif()
 
 find_package(yaml-cpp CONFIG REQUIRED QUIET)
+if(NOT yaml-cpp_FOUND)
```

Comment:
How can this ever happen with the previous line passing `REQUIRED`?

Reply:
You're absolutely right. Addressed in https://github.com/ros2/yaml_cpp_vendor/pull/9/commits/8edc9651016366203e6b1bcc117244f7f497bdfe